index.css

```diff
@@ -1,281 +1,73 @@
-:root {
-    /* colors */
-    --clr-dark: 230 35% 7%;
-    --clr-light: 231 77% 90%;
-    --clr-white: 0 0% 100%;
-    --clr-gold: 235 216% 212%;
-    --clr-RoseGold: 351 34% 57%;
-    --clr-White-Gold: 31 25% 65%;
-    --clr-vintage-Red:4 66% 28%;
-    --clr-wood-Brown:42 37% 41%;
-    --clr-green:145, 63%, 49%;
-    
-    /* font-sizes */
-    --fs-900: 9.375rem;
-    --fs-800: 6.25rem;
-    --fs-700: 3.5rem;
-    --fs-600: 2rem;
-    --fs-500: 1.75rem;
-    --fs-400: 1.125rem;
-    --fs-300: 1rem;
-    --fs-200: 0.875rem;
-}
-body{
-    background-color: hsl(var(--clr-dark));
-}
-.navigation{
-    height: 65px;
-    display:grid;
-    background-color: hsl(var(--clr-dark));
-    grid-template-columns: 1fr,1fr,1fr,1fr,1fr,1fr,1fr,1fr,1fr;

```

Comment:
- [ ]  There seems to be empty columns that are here to space the elements inside the navigation. 
Instead of using this bad practice, There should be other grid or flex properties to space the elements apart.